.travis.yml

```diff
@@ -10,15 +10,19 @@ cache:
     - $HOME/.cache/pip
 
 install:
-  - pip install "$DJANGO"
+  - pip install "$DEPENDENCIES"
   - pip install -r requirements-dev.txt
 
 env:
   matrix:
-   - DJANGO="Django<1.9"
-   - DJANGO="Django<1.10"
-   - DJANGO="Django<1.11"
-   - DJANGO="Django<1.12"
+   - DEPENDENCIES="Django<1.9 django-oauth-toolkit<1.0.0"
```

Comment:
Why not `==` instead of `<` ?

Reply:
I was trying to change the minimum possible for my PR to be approved, but that's breaking the builds right now. Gonna see if I can change to `==` and make it work.

Reply:
@drgarcia1986 with `<` we get the last version of X

| Query  | Version |
| - | - |
| `<1.9` | `1.8.LAST`  |
|`<1.10` | `1.9.LAST`  |
| `<1.11` | `1.10.LAST`  |
| `<1.12`  | `1.11.LAST`  |

Reply:
@jairhenrique if we put `==1.8` won't it always install the version `1.8.LAST` ?

Reply:
@jarussi  no.

```bash
pip install Django==1.8
Collecting Django==1.8
  Downloading Django-1.8-py2.py3-none-any.whl (6.2MB)
```

```bash
pip install 'Django<1.9'
Collecting Django<1.9
  Downloading Django-1.8.18-py2.py3-none-any.whl (6.2MB)
```